internal/docstore/mongodocstore/localmongo.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Go Cloud Development Kit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Runs only tests relevant to the current pull request.
+# At the moment, this only gates running the Wire test suite.
```

Comment:
This comment seems copy+pasted from somewhere else and not relevant here (and elsewhere).

Further, its original comment seems obsolete now?

Reply:
Yes, copypasta. Updated in the new scripts, and also updated the one that I copy-pasted from, which as you point out was also stale.